picfixPy/vibrance.py

```diff
@@ -1,20 +1,54 @@
-def vibrance(input_img, intensity=5, output_img):
-    
+def vibrance(input_img, intensity=0, display = False, output = ''):
```

Comment:
Good idea for thinking of the `display` argument, I will add this argument to my function as well.